library/std/convert.qs

```diff
@@ -264,4 +264,20 @@ namespace Microsoft.Quantum.Convert {
         );
     }
 
+    /// # Summary
+    /// Produces string representation of given double with desired precision (performs rounding).
+    ///
+    /// # Input
+    /// ## input
+    /// Double to be converted.
+    /// ## precision
+    /// Number of decimal digits.
+    /// # Example
+    /// ```qsharp
+    /// Message($"{DoubleAsStringWithPrecision(0.354, 2)}"); // Prints 0.35
```

Comment:
Since the examples are the only source of information for the reader on padding with zeroes and corner cases such as rounding and precision=0, I'd recommend including samples for padding, precision=0, and rounding, one sample each

Reply:
As in add those cases to the # Example section?

Reply:
Yes.

Reply:
You can just add them as lines to the same Q# code snippet